.github/workflows/screenshots.yml

```diff
@@ -13,6 +13,9 @@ jobs:
     name: Send screenshots
     runs-on: ubuntu-latest
 
+    # Skip when secrets are unavailable on forks
+    if: github.repository == 'alphagovuk/govuk-frontend'
```

Comment:
Should this be `alphagovuk` or `alphagov`?

Reply:
Suppose that's why it skipped successfully 🤦‍♂️ 

Thanks 😆

Reply:
Typo fixed. I'll carry on testing